Jenkinsfile

```diff
@@ -97,9 +97,6 @@ pipeline {
                 }
             }
             steps {
-                // quick check that no files are modified for the go modules
```

Comment:
I just removed a single instance of this check, and voluntary let the release and master build with it. Let me know if we should remove the others too.

Reply:
@travisn  ^

Reply:
Looks good to remove this one. For the others, we really want to prevent ever publishing a build that has the `dirty` tag, which happens when any local files are modified. Let's keep those to help with debugging in case a modified file does sneak in.